acir/src/circuit/opcodes/black_box_function_call.rs

```diff
@@ -76,6 +76,16 @@ pub enum BlackBoxFuncCall {
         inputs: Vec<FunctionInput>,
         outputs: Vec<Witness>,
     },
+    VerifyProof {
```

Comment:
I don't think VerifyProof is the right name for this opcode, since that would indicate that this should return a boolean as output and or at the least that once this opcode was called, that the proof was verified

Reply:
Hmm yes good point. We could call it `VerifyRecursiveProof` or `RecursiveVerification` (although I do `VerifyProof` implies this as well)? I will leave comments indicating the purpose of the input and output aggregation object

Reply:
Hmm it seems to be aggregating and not entirely verifying

Reply:
We do verify the previous aggregation state technically, and then aggregate the current proof onto it. So aggregation may be a better name. `RecursiveAggregation`?

Reply:
I changed the name to `RecursiveAggregation`